simpleclient_hotspot/src/main/java/io/prometheus/client/hotspot/SafepointExports.java

```diff
@@ -0,0 +1,61 @@
+package io.prometheus.client.hotspot;
+
+import io.prometheus.client.Collector;
+import io.prometheus.client.CounterMetricFamily;
+import sun.management.HotspotRuntimeMBean;
+import sun.management.ManagementFactoryHelper;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Exports metrics about JVM Safepoints.
+ * <p>
+ * Example usage:
+ * <pre>
+ * {@code
+ *   new SafepointExports().register();
+ * }
+ * </pre>
+ * Example metrics being exported:
+ * <pre>
+ *   jvm_safepoint_count{} 200
+ *   jvm_safepoint_total_time_seconds{} 6.7
+ *   jvm_safepoint_sync_time_seconds{} 6.7
+ * </pre>
+ */
+public class SafepointExports extends Collector {
+  private final HotspotRuntimeMBean hotspotRuntimeMBean;
+
+  public SafepointExports() {
+    this(ManagementFactoryHelper.getHotspotRuntimeMBean());
+  }
+
+  SafepointExports(HotspotRuntimeMBean hotspotRuntimeMBean) {
+    this.hotspotRuntimeMBean = hotspotRuntimeMBean;
+  }
+
+  public List<MetricFamilySamples> collect() {
+    CounterMetricFamily safepointCount = new CounterMetricFamily(
+        "jvm_safepoint_count",
+        "The number of safepoints taken place since the JVM started.",
+            hotspotRuntimeMBean.getSafepointCount());
+
+    CounterMetricFamily safepointTime = new CounterMetricFamily(
+        "jvm_safepoint_total_time_seconds",
```

Comment:
The total here is a little confusing, I'd remove it.

Might these two make more sense as one Summary?